dll/win32/shell32/stubs.cpp

```diff
@@ -25,15 +25,34 @@ ShortSizeFormatW(LONGLONG llNumber)
     return NULL;
 }
 
-/*
- * Unimplemented
+/*************************************************************************
+ *  SHFindComputer [SHELL32.91]
+ *
+ * Invokes the shell search in My Computer. Used in SHFindFiles.
+ * Two parameters are ignored.
  */
 EXTERN_C BOOL
 WINAPI
-SHFindComputer(LPCITEMIDLIST pidl1, LPCITEMIDLIST pidl2)
+SHFindComputer(LPCITEMIDLIST pidlRoot, LPCITEMIDLIST pidlSavedSearch)
 {
-    FIXME("SHFindComputer() stub\n");
-    return FALSE;
+    UNREFERENCED_PARAMETER(pidlRoot);
+    UNREFERENCED_PARAMETER(pidlSavedSearch);
+
+    TRACE("%p %p\n", pidlRoot, pidlSavedSearch);
+
+    IContextMenu *pCM;
+    HRESULT hr = CoCreateInstance(CLSID_ShellSearchExt, 0, CLSCTX_INPROC_SERVER,
+                                  IID_IContextMenu, (void **)&pCM);
+    if (FAILED_UNEXPECTEDLY(hr))
+        return hr;
+
+    CMINVOKECOMMANDINFO InvokeInfo = { sizeof(InvokeInfo) };
+    InvokeInfo.lpParameters = "{996E1EB1-B524-11D1-9120-00A0C98BA67D}";
```

Comment:
It would be interesting to know what this GUID corresponds to.
Also since I see that we have some code already that calls this SHFindComputer function, have you observed some new behaviour now?

Reply:
Considered nothing in ReactOS uses that guid, I doubt anything new happens.

Reply:
In Windows, this CLSID is defined in `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\FindExtensions\Static\ShellSearch\1\SearchGUID`.

Reply:
Found that GUID in `browseui.dll` and `shell32.dll`:
![image](https://github.com/reactos/reactos/assets/2107452/416d3ae1-fb02-4952-bacf-0bb1c6391249)

Reply:
> In Windows, this GUID is defined in `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\FindExtensions\Static\ShellSearch\1\SearchGUID`.

Probably as part of shell search modules registered by GUID (I doubt we have that functionality implemented), that specific GUID here would be the one used to "find computers"...